scout/models/case/case_loading_models.py

```diff
@@ -381,7 +381,7 @@ class CaseLoader(BaseModel):
     gene_fusion_report: Optional[str] = None
     gene_fusion_report_research: Optional[str] = None
     gene_panels: Optional[List[str]] = []
-    genome_build: Union[str, int] = Field(38, alias="human_genome_build")
+    genome_build: str
```

Comment:
I don't follow: wouldn't this drop the "human_genome_build" completely? Not that we use it locally, but it did exist? Perhaps deprecate first?

Reply:
Nono, it's still used, check the validation down, it collects the build from wither genome_build or human_genome_build